core/src/main/java/tachyon/worker/WorkerStorage.java

```diff
@@ -661,6 +681,9 @@ public Command heartbeat() throws IOException {
    */
   public void initializeStorageTier() throws IOException {
     mStorageTiers = new ArrayList<StorageTier>(WorkerConf.get().STORAGE_LEVELS);
+    for (int k = 0; k < StorageLevelAlias.values().length; k ++) {
+      mCapacityBytesByAlias.add((long) 0);
```

Comment:
mCapacityBytesByAlias is not needed, the capacity and used bytes can be gotten from StorageTier